src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -1280,19 +1280,20 @@ public static IEnumerable<object[]> Sqrt_TestData()
             // Simple known values.
             yield return new object[] { 0, 0, 0, 0 };
             yield return new object[] { 1, 0, 1, 0 };
-            yield return new object[] { -1, 0, 0, 1 };
             yield return new object[] { 0, 1, 0.707106781186547, 0.707106781186547 };
             yield return new object[] { 0, -1, 0.707106781186547, -0.707106781186547 };
 
             // Extreme values don't overflow, even when intermediate quantities would if handled naively.
             yield return new object[] { double.MaxValue, 0.0, Math.Sqrt(double.MaxValue), 0.0 };
-            yield return new object[] { -double.MaxValue, 0.0, 0.0, Math.Sqrt(double.MaxValue) };
             yield return new object[] { 0, double.MaxValue, 9.48075190810917E+153, 9.48075190810917E+153 };
             yield return new object[] { 0, double.MinValue, 9.48075190810917E+153, -9.48075190810917E+153 };
         }
 
         public static IEnumerable<object> Sqrt_AdvancedTestData ()
```

Comment:
The "Advanced" set of test data is used everywhere except for .NET Framework.

https://github.com/dotnet/corefx/pull/17170/files/5ba5a836f31b24723d1e985da7c8c13a8a66e69e#diff-19973eb77d5f57cf6c8661e7fe588dc9R1334